buddy_init/init_project.py

```diff
@@ -0,0 +1,55 @@
+from functools import partial
+
+import argparse
+
+from .init_actions import *
+from .init_framework import WorkingDirectory, prompt, add_action_toggle
+
+
+def init(args=None):
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--yes', '-y', help='Forces all boolean options to true', action='store_true', default=None)
+    parser.add_argument('--dest', '-d', help='The destination of your epic project! :D', required=True)
+    parser.add_argument('--force', '-f', help='Starts the project from scratch', action='store_true', default=False)
```

Comment:
add that this will remove all files in the directory.

Reply:
`-d` does not accept `.` can we do something about it?

Reply:
actually it doesn't accept a path at all, do you want to have this behaviour?

Reply:
On a side note:
```cd ~
buddy-init -f --dest .```
This is may be worse than tragic

Reply:
+1 remove -f

Reply:
Luke, [don't] use the force!... Removed